modules/ROOT/pages/migrate-from-an-earlier-version-of-bonita-bpm.adoc

```diff
@@ -1,126 +1,132 @@
-* REST API Extensions: These might continue to work in the new version, but should be tested depending on your custom code.
+* Bonita 7.0 only supports JRE version 7. If updating to Bonita 7.0 to 7.4.x, please update your JRE to version 7.
+* Bonita 7.5 only supports JRE version 8. If updating to Bonita 7.5 to Bonita 2021.1-0811, please update your JRE to version 8.
+* Bonita 2021.2 only supports JRE version 11. If updating to Bonita 2021.2 or greater, please update your JRE to version 11.
 
-When the script has finished,
-you need to complete the migration by unzipping and configuring a bundle for the new version.
-See  <<migrate,Migrate your platform>> for step-by-step instructions.
+For more information, go to the Customer Service Center and read the Support Guide and Supported Environment Matrix for Server.
+
+[#rdbms_requirements]
 
-== Constraints
+=== Database
+* The migration script supports MySQL, Postgres, Oracle, and Microsoft SQL Server. *There is no migration for h2 database*.
+* The target Bonita version may not support the *version of the database* used with your source Bonita version. You may then need to update the version of your database prior to running the migration tool. To make sure:
+ ** Please check the xref:hardware-and-software-requirements.adoc[database requirements].
+ ** If you need to update your database, make sure to apply all the xref:database-configuration.adoc#specific_database_configuration[RDBMS customisations required by Bonita] when setting up the new version.
+* If you have added *custom indexes* to certain tables in the engine database, you must *remove them before lauching* the update procedure. If you do not remove these indexes, the update procedure will not complete.
 
-* If you have added custom indexes to certain tables in the Engine database, you must remove them before migrating to a later version. If you do not remove these indexes, the migration will not complete.
-* There is no guarantee that the Look & Feel definition is compatible across maintenance versions.
-For example, in 6.2.2, `jquery+` was renamed `jqueryplus` in `BonitaConsole.html`, for compatibility with more application servers.
-If you are using a custom Look & Feel,  xref:managing-look-feel.adoc[export] it before migrating.
-Then after the migration is complete,  xref:managing-look-feel.adoc[export the default Look & Feel] from the new version,
-modify your custom Look & Feel to be compatible with the new definition, and with the  xref:creating-a-new-look-feel.adoc[recommendations for form footers].
-Then  xref:managing-look-feel.adoc[import] your updated custom Look & Feel into Bonita Portal.
-* The migration script supports MySQL, Postgres, Oracle, and Microsoft SQL Server. There is no migration for h2 database.
+=== Custom Portal Look & Feel
+*For target version older than Bonita 2021.2 only*: there is no guarantee that the Portal Look & Feel definition is compatible across maintenance versions.
```

Comment:
Well just write this on older documentation version and then remove it from here... 
This is typically the can of noise that make the documentation unpleasant to read.

Reply:
removed